### PR TITLE
add final text

### PR DESCRIPTION
**Pull request by marte1337**
final version of the README file

### CODE REVIEW

**Comment by Wizzard26** [comment]
Nice list

File: README.md [hunk 1]
```diff
@@ -1,25 +1,14 @@
...
-Create a new branch with git switch -c <branchname>.
-Make changes to the code / write your code fpr the feature.
-Push the changes and the new branch with git push -u origin <branchname> (after you have done this once you can use git push for this branch)
-Create a pull request on GitHub from the new branch into main
-Share the pull request with your team
-Review the pull request, implement changes if needed, push again to update the pull request until it gets approved
-Merge the pull request into main
-Don't forget to git pull inside the main branch on your local machine
-Delete the new branch on GitHub and locally
+| command                      | functionality                        |
```
Comment (aaaleeenaaa) [inline README.md:8]: Nice list!

**Review by aaaleeenaaa — APPROVED** [review]
Looks really good!

File: README.md (hunk 1 above)
Comment (Leukipos) [reply README.md:8]: this is amazing...

**Review by Leukipos — COMMENTED** [review]
a lot of informations... fabulous

**Pull request merged**